core/generated-docs/core/ecsql-common/index.ts

```diff
@@ -0,0 +1,5 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
why is this here?

Reply:
was probably generated by the `builddocsite` command that I ran locally. I thought it was needed, but I see that it couldn't be. I deleted the files.